frontend_app_admin_workspace_user/src/component/AdminUser.jsx

```diff
@@ -375,6 +438,66 @@ export class AdminUser extends React.Component {
             )}
           </div>
 
+          {state.popupInfos && (
+            <CardPopup
+              customClass='adminUser___right'
+              customColor={props.config.hexcolor}
+              faIcon='far fa-id-card'
+              label={props.t('View user limitations')}
+              onClose={this.handleTogglePopupInfo}
+            >
+              <div>
+
+                {props.config.limitation__max_non_guest_users !== -1 && (
+                  <p>
+                    {
+                      props.t('Normal users: {{currentGuests}}/{{maxGuests}} ({{count}} slots remaining)',
+                        {
+                          currentGuests: this.getNumberActiveUsers(),
+                          maxGuests: props.config.limitation__max_non_guest_users,
+                          count: props.config.limitation__max_non_guest_users - this.getNumberActiveUsers()
+                        }
+                      )
+                    }
+                  </p>
+                )}
+                {props.config.limitation__max_non_guest_users === -1 && (
+                  <p>{props.t('No limit for the number of users')}</p>
+                )}
+                {props.config.limitation__max_guest_users !== -1 && (
+                  <p>
+                    {
+                      props.t('Guest users: {{currentGuests}}/{{maxGuests}} ({{count}} slots remaining)',
+                        {
+                          currentGuests: this.getNumberActiveGuests(),
+                          maxGuests: props.config.limitation__max_guest_users,
+                          count: props.config.limitation__max_guest_users - this.getNumberActiveGuests()
+                        }
+                      )
+                    }
+                  </p>
+                )}
+                {props.config.limitation__max_guest_users === -1 && (
+                  <p>{props.t('No limit for the number of guests')}</p>
+                )}
+
+                {
+                  props.config.limitation__max_guest_user_nb_space !== -1 && (
+                    <p>
+                      {props.t(
+                        'A guest can join up to {{count}} space',
+                        { count: props.config.limitation__max_guest_user_nb_space })}
+                    </p>
+                  )
+                }
+                {props.config.limitation__max_guest_user_nb_space === -1 && (
+                  <p>{props.t('No limit to the number space a guest user can join')}</p>
```

Comment:
harmonize the usage of whether "no limit to" or "no limit for".
I'm sure which one is better